mobile/examples/React_Native/ort-rn-basic-usage/README.md

```diff
@@ -0,0 +1,109 @@
+# ONNX Runtime React Native Expo Example Application
+
+This is a [React Native](https://reactnative.dev/docs/getting-started) application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) via [Expo](https://docs.expo.dev/) platform. The demo app demonstrates how to accomplish simple tasks such as loading onnx models and creating inference sessions, etc. in a react native expo application.
+
+## Prerequisites
+
+1. Install [Node.js](https://nodejs.org/en)
+2. Install [Expo-CLI](https://docs.expo.dev/more/expo-cli/)
+    ```sh
+    npm install -g expo-cli
+    ```
+3. Install [Yarn](https://classic.yarnpkg.com/en/docs/install#mac-stable) (Recommended)
+    ```sh
+    npm install -g yarn
+    ```
+**NOTE:**
+   For creating a new react native expo project from scratch, refer to: https://docs.expo.dev/get-started/create-a-project/
+
+## Set up
+
+1. Run `yarn install` to set up JavaScript dependencies.
+    ```sh
+    yarn install
+    ```
+2. Install NPM `onnxruntime-react-native` package.
+    ```sh
+    expo install onnxruntime-react-native@latest
+    ```
+   Note: Recommend to install the current latest release version of ORT react native npm package. Can also update to install dev version npm package if needed.
+
+
+   [Link to npm `onnxruntime-react-native` packages](https://www.npmjs.com/package/onnxruntime-react-native?activeTab=versions)
+3. Prepare the model.
+
+    -  Model files are usually placed under `<PROJECT_ROOT>/assets`.
+    
+       In this sample application, a simple ORT format MNIST model (`mnist.ort`) is provided.
+
+    -  File `metro.config.js` under `<PROJECT_ROOT>` adds the extension `ort` to the bundler's asset extension list, which allows the bundler to include the model into assets.
+       
+       `metro.config.js` file in this sample application looks like:
+
+       ```js
+       const { getDefaultConfig } = require('@expo/metro-config');
+       const defaultConfig = getDefaultConfig(__dirname);
+       defaultConfig.resolver.assetExts.push('ort');
+       module.exports = defaultConfig;
+       ```
+       Adjust the extension to `.onnx` accordingly if you are loading a ONNX model in your application.
```

Comment:
Update to default to .onnx model. I don't think we need to mention .ORT format models here anymore.